pkg/crc/cluster/cluster.go

```diff
@@ -198,3 +219,47 @@ func addPullSecretToInstanceDisk(sshRunner *ssh.SSHRunner, pullSec string) error
 	}
 	return nil
 }
+
+func StartKubeApiserverOperator(oc oc.OcConfig) error {
+	cmdArgs := []string{"scale", "--replicas=1", "deployment", "--all", "-n", "openshift-kube-apiserver-operator"}
+
+	if err := oc.WaitForOpenshiftResource("deployment"); err != nil {
+		return err
+	}
+
+	if _, stderr, err := oc.RunOcCommand(cmdArgs...); err != nil {
+		return fmt.Errorf("Failed to start Openshift apiserver  %v: %s", err, stderr)
+	}
+	return nil
+}
+
+func WaitforRequestHeaderClientCaFile(oc oc.OcConfig) error {
+	cmdArgs := []string{"get", "cm/extension-apiserver-authentication", `-ojsonpath={.data.requestheader-client-ca-file}`,
```

Comment:
It might be possible to use `oc observe configmap -n kube-system` to watch for the changes, but seems a bit tricky to do properly.

Reply:
@cfergeau yes looks like if we use `oc observe` we need to have a script associate with it which will make it bit complicated then just query ourself and see if that resource is updated.